### PR TITLE
Updated CapRover <> Hetzner Docs

### PR DESCRIPTION
**Pull request by jtrugman**
Hi Hetzner Team,

When setting up CapRover on my new Hetzner instance I noticed that the current docs about installing CapRover on Hetzner Cloud are a bit out of date. After successfully installing and running CapRover on my new instance, I am submitting a PR for updating the docs to be more current with how CapRover is installed and ran.

Please let me know any feedbacks you may have.

Thanks

### CODE REVIEW

**Comment by svenja11** [comment]
Thank you for your contribution @jtrugman! The author information at the top of the file is automatically displayed at the top of the tutorial at [community.hetzner.com](https://community.hetzner.com/). However, this only works for one author, so you will have to remove one of the authors.

Since your changes are rather minor, I would suggest that you remove your own name. If you want to, you can add a hint like this right above the introduction:

````
> This tutorial was updated by jtrugman
````

**Comment by svenja11** [comment]
Please add this text as a comment to this pull request and replace `[submitter's name and email address here]` with your own information:


> I have read and understood the Contributor's Certificate of Origin available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it. Signed-off-by: [submitter's name and email address here]

File: tutorials/install-caprover/01.en.md [hunk 1]
```diff
@@ -58,6 +60,8 @@ Go to your domain's DNS management panel and create a new `A` record pointing to
 
 To verify that you correctly configured your domain, visit <https://mxtoolbox.com/DNSLookup.aspx>, input `foobar.<your_domain.com>` (`foobar.<subdomain_name>.<your_domain.com>` if you used a subdomain) and check if the domain resolves to the IP address you configured in your DNS settings. **DNS propagation can take a few minutes, so if it doesn't work, just wait some more.** It should take under 30 minutes for DNS to fully propagate.
 
+> **Note:** It is critical that a wild card `*` is assigned to the `HOST` in the `A` record. This allows Caprover to create unique addresses for your application(s) running on the server
```
Comment (svenja11) [inline tutorials/install-caprover/01.en.md:63]:
```suggestion
> **Note:** It is critical that a wild card `*` is assigned to the `HOST` in the `A` record. This allows CapRover to create unique addresses for your application(s) running on the server.
```

**Comment by jtrugman** [comment]
I have read and understood the Contributor's Certificate of Origin available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it. Signed-off-by: Justin Trugman, jtrugman14@gmail.com [Date Effective July 15th, 2023]

**Comment by jtrugman** [comment]
Apologies for the delay on this, thanks for your feedbacks @svenja11.
Just implemented them.

**Comment by svenja11** [comment]
Looks good now, thank you for the update!

**Pull request merged**